go-controller/pkg/ovn/gateway_cleanup.go

```diff
@@ -26,7 +28,7 @@ func (oc *DefaultNetworkController) gatewayCleanup(nodeName string) error {
 	var nextHops []net.IP
 
 	gwIPAddrs, err := util.GetLRPAddrs(oc.nbClient, types.GWRouterToJoinSwitchPrefix+gatewayRouter)
-	if err != nil {
+	if err != nil && !errors.Is(err, libovsdbclient.ErrNotFound) {
```

Comment:
ouch! we should have continued instead of returning when `errNotFound`

Reply:
nvm: https://github.com/ovn-org/ovn-kubernetes/commit/25d892cc9e318d19738d9519e0e20fbab28f7eae